test/integration/features/addons/addon-registry-route.feature

```diff
@@ -57,17 +57,22 @@ so that user can able to push their container images directly to registry and de
      Then with up to "10" retries with wait period of "500ms" the "status code" of HTTP request to "/" of service "nginx" in namespace "myproject" is equal to "200"
       And with up to "10" retries with wait period of "500ms" the "body" of HTTP request to "/" of service "nginx" in namespace "myproject" contains "Welcome to nginx!"
 
-  Scenario: User restarts Minishift
+  Scenario: User can login to OpenShift integrated docker registry after Minishift restart
     Given Minishift has state "Running"
      When executing "minishift stop" succeeds
      Then executing "minishift start" succeeds
+      And Minishift should have state "Running"
+     Then with up to "20" retries with wait period of "3" seconds container name "registry_docker-registry" should be "running"
```

Comment:
Please put this and following steps into a separate scenario. Scenario: Restart, Scenario: User can login to OpenShift integrated docker registry